libcxx/include/__algorithm/stable_sort.h

```diff
@@ -253,6 +253,12 @@ _LIBCPP_CONSTEXPR_SINCE_CXX26 void __stable_sort(
   if constexpr (__allowed_radix_sort) {
     if (__len <= __buff_size && __len >= static_cast<difference_type>(__radix_sort_min_bound<value_type>()) &&
         __len <= static_cast<difference_type>(__radix_sort_max_bound<value_type>())) {
+      for (auto* __p = __buff; __p < __buff + __buff_size; ++__p) {
+        std::__construct_at(__p, 0);
+      }
+      __destruct_n __d(__buff_size);
+      unique_ptr<value_type, __destruct_n&> __h2(__buff, __d);
```

Comment:
This is all only required for `constexpr` support, right? I'm pretty sure you can drop the destruction, since the elements should always be trivial. For the construction, I think we don't need the zero and we should probably only do it during constant evaluation.

Reply:
Yes, you're right.
Done all three things.